src/views/Header.js

```diff
@@ -72,6 +78,10 @@ class Header extends React.Component<void, HeaderProps, void> {
 
   props: HeaderProps;
 
+  state = {
+    widths: {},
+  };
+
   shouldComponentUpdate(nextProps: HeaderProps, nextState: any): boolean {
```

Comment:
Should switch this to `PureComponent` and remove this `shouldComponentUpdate` block